src/test/java/calculator/StringCalculatorTest.java

```diff
@@ -0,0 +1,49 @@
+package calculator;
+
+import static calculator.StringCalculator.*;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+class StringCalculatorTest {
+
+	@DisplayName("빈 문자열 또는 null 값인 경우 0을 반환한다")
+	@Test
+	void NullOrBlank() {
+		assertThat(cal(null)).isEqualTo(0);
+		assertThat(cal("")).isEqualTo(0);
+	}
```

Comment:
```suggestion
	@DisplayName("빈 문자열 또는 null 값인 경우 0을 반환한다")
	@ParameterizedTest(name = "입력값: {0}")
	@NullAndEmptySource
	void NullOrBlank(String input) {
		assertThat(cal(input)).isEqualTo(0);
	}
```
ParameterizedTest를 사용하면 중복코드를 제거할 수 있습니다.